dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaExtension.kt

```diff
@@ -153,6 +148,9 @@ constructor(
      * }
      * ```
      *
+     * _Aside: Launching [without isolation][WorkerExecutor.noIsolation] is not an option, because
```

Comment:
not sure it's needed here, as it's not actionable (e.g why I need to know this, why it's required, what is it, etc)
probably we can safely remove this part